sig-and-wg/sig-core/proposals/central-connector-service.md

```diff
@@ -0,0 +1,133 @@
+A connected Application calls this endpoint periodically and checks the cluster status. 
+
+### Certificate revocation 
+
+The client certificates and intermediate server certificates must be revoked as soon as they are compromised. The list of revoked certificates will be stored in the central Connector Service and synchronized with all Kyma clusters. The list will contain both the client certificates and the server intermediate certificates.
+
+The fingerprint of the compromised certificate will be added to the Application Connector. It will ensure that the connected Application which uses the compromised certificates can no longer perform any calls.
+
+>**NOTE:** The current versions of Istio and the Nginx-controller do not support the revocation list. The further plan to add support for this feature will be provided later.
+
+## Proof of Concept
+
+### Prerequisites:
+
+- Private key (`rootCA.key`) and certificate (`rootCA.crt`) generated as a root CA
+- Kyma cluster provisioned with `rootCA.key` and `rootCA.crt` as a CA
+- Application `test-application` created
+
+### Steps
+
+1. Generate the client certificate (`client.crt`) for the `test-application` using the Connector Service.
+2. Access the cluster using the generated `client.crt`
+
+    ```
+    curl  https://gateway.{CLUSTER_DOMAIN}/test-application/v1/metadata/services --cert client.crt --key client.key
+    ```
+
+3. Generate the intermediate CA signed with the root CA
+
+    ```
+    openssl genrsa -out intermediate.key 4096
+    openssl req -new -out intermediate.csr -key intermediate.key -subj /CN="intermediate"
+    openssl x509 -req -sha256 -in intermediate.csr -out intermediate.crt -CAkey rootCA.key -CA rootCA.crt -days 1800 -CAcreateserial -CAserial serial
+    ```
+
+4.  Create the certificate chain containing `rootCA.crt` and `intermediate.crt`.
+
+    ```
+    cat rootCA.crt intermediate.crt > intermediate-chain.crt
+    ```
+
+5. Edit the secret containing CA `nginx-auth-ca` to use `intermediate-chain.crt` as `ca.crt` and `intermediate.key` as `ca.key`.
+
+    ```
+    export CERT=$(cat intermediate-chain.crt | base64)
+    export KEY=$(cat intermediate.key | base64)
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    data:
+      ca.crt: $CERT
+      ca.key: $KEY
+    kind: Secret
+    metadata:
+      name: nginx-auth-ca
+      namespace: kyma-integration
+    type: Opaque
+    EOF
+    kubectl -n kyma-system delete po -l "app=nginx-ingress"
+    ```
+
+6. Wait for the Pod to restart. Access the cluster using the previously generated `client.crt`:
+
+    ```
+    curl  https://gateway.{CLUSTER_DOMAIN}/test-application/v1/metadata/services --cert client.crt --key client.key
+    ```
+
+7. Revert changes by setting the root CA-signed certificate as the original one and call the cluster using of the intermediate certificate as the client certificate.
```

Comment:
I get the gist of it, but maybe it'd be better to show what commands to execute this action. Split into two steps to make it clearer if necessary.